source/PlayerInfo.h

```diff
@@ -307,6 +307,7 @@ class PlayerInfo {
 	bool SelectShips(const Rectangle &box, bool hasShift);
 	bool SelectShips(const std::vector<const Ship *> &stack, bool hasShift);
 	void SelectShip(const Ship *ship, bool hasShift);
+	void UnselectShip(const std::shared_ptr<Ship> ship);
```

Comment:
I think we usually use "deselect" to mean "remove from the list of selected items", for example, in `InfoPanelState`.